src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -25,7 +25,7 @@ object Run  {
 
   def apply(pipeline: Pipeline): Run = {
     val rpr = new RunPipelineRequest().setPipelineId(pipeline.id).setProjectId(pipeline.projectId).setServiceAccount(JesServiceAccount)
-    val tag = s"JES Run [UUID(${pipeline.workflow.shortId}):${pipeline.key.scope.name}]"
+    val tag: String = s"JES Run [UUID(${pipeline.workflow.shortId}):${pipeline.key.tag}]"
```

Comment:
`val tag` instead of `val tag: String`

I go back and forth about explicitly typing methods, but I generally don't use them for local val/var unless required for some reason.